### ISSUE TITLE
"change the current shell" exercise

### ISSUE THREAD

**Issue by cochrand** [ReproNim/module-reproducible-basics#24]
So, I'm using a Windows computer.  I can see that I'm running the bash shell.  However, tcsh is not installed, so I can not start it.  It would be helpful if there was a command to tell me what shells I have access to on a given computer, and if you really want me to use tcsh, how to get it if I don't have it.

**Comment by yarikoptic** [comment]
good point -- so people might not be able to do the exercise if there is no alternative shell available... I guess we should note that.

And good question about available shells - I could only offer doing  `ls /bin/*sh` since all shells I know/we mention are named ending with `sh`, so that simple `ls` should be sufficient.  Not sure though if there is any other shell on windows, but if it is the 'ubuntu based bash shell', you should be able to `apt-get install tcsh`
